charts/consul/templates/webhook-cert-manager-configmap.yaml

```diff
@@ -17,10 +17,10 @@ data:
       {
         "name": "{{ template "consul.fullname" . }}-connect-injector-cfg",
         "tlsAutoHosts": [
-          "{{ template "consul.fullname" . }}-connect-injector-svc",
-          "{{ template "consul.fullname" . }}-connect-injector-svc.{{ .Release.Namespace }}",
-          "{{ template "consul.fullname" . }}-connect-injector-svc.{{ .Release.Namespace }}.svc",
-          "{{ template "consul.fullname" . }}-connect-injector-svc.{{ .Release.Namespace }}.svc.cluster.local"
+          "{{ template "consul.fullname" . }}-connect-injector",
+          "{{ template "consul.fullname" . }}-connect-injector.{{ .Release.Namespace }}",
+          "{{ template "consul.fullname" . }}-connect-injector.{{ .Release.Namespace }}.svc",
+          "{{ template "consul.fullname" . }}-connect-injector.{{ .Release.Namespace }}.svc.cluster.local"
```

Comment:
need to think about what will happen during an upgrade with these certs and if there will be any downtime. Might be safer to keep this one as is.

Reply:
Wouldn't the upgrade cause both, the connect container as well and the cert-manager container to get recreated, in which case the upgrade should occur without downtime? Am I missing something?

Reply:
right so I guess:
- before upgrade svc and certs match the mutatingwebhookconfig
- upgrade modifies mutatingwebhookconfig to point at new svc object
- new svc has no pods yet (restarting) but that's okay
- inject pods and cert manager start
- cert manager mints new cert, updates mutatingwebhook config and inject pods pick up new secret

So it should all be fine, just a bit of downtime as those new connect inject pods start.

Reply:
yup! i expect it to be the same amount of downtime as any usual `upgrade`. Maybe a touch longer? But it should be fine.